src/server/index.tsx

```diff
@@ -65,18 +67,34 @@ export const createServer = ({
   };
 
   const cdnURL = env.PUBLIC_URL || "'none'";
+  const connectSrc = [
+    // Relax the connect-src if using the proxy otherwise
+    // Use the env var or 'none' if the API host isn't set.
+    env.REACT_APP_USE_INSECURE_PROXY === 'true'
+      ? "'self'"
+      : env.REACT_APP_API_HOST || "'none'",
+  ];
+
+  if (env.REACT_APP_SENTRY_DSN) {
+    try {
+      const { host, protocol } = url.parse(env.REACT_APP_SENTRY_DSN);
+
+      if (host && protocol) {
+        connectSrc.push(`${protocol}//${host}`);
+      }
+    } catch {
+      // eslint-disable-next-line no-console
+      console.error(
```

Comment:
typo'ed the url will be hard to track, this fails when we don't pass a string to `url.parse()`...

Reply:
also, this should probably not be handled here but in `configureApplication()` because the server will not use sentry.

Reply:
Thinking about it, that would break everything once deployed, no? If the server does not start, it won't prevent the deployment to finish but we'll end up with a blank page (or a 503 from nginx).

I think we should make sure sentry is configured correctly, but I am not sure this is the right place. all I wanted here is not hard-code our mozilla sentry host.